test.sh

```diff
@@ -108,8 +161,6 @@ function initializeCert() {
      -keystore $JRE_HOME/lib/security/cacerts \
      -file /test-cert/conjur-cucumber.der -storepass changeit"
   docker exec ${conjur_test_cid} ${import_command}
```

Comment:
it looks like the command to import the certs is no longer valid - you can see that in the build logs here:
```
import cert inside test https container
[2019-05-13T12:14:15.051Z] Certificate was added to keystore
[2019-05-13T12:14:15.051Z] [Storing /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/security/cacerts]
[2019-05-13T12:14:15.051Z] keytool error: java.io.FileNotFoundException: /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/security/cacerts (No such file or directory)
[2019-05-13T12:14:15.051Z] java.io.FileNotFoundException: /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/security/cacerts (No such file or directory)
[2019-05-13T12:14:15.051Z] 	at java.io.FileOutputStream.open0(Native Method)
[2019-05-13T12:14:15.051Z] 	at java.io.FileOutputStream.open(FileOutputStream.java:270)
[2019-05-13T12:14:15.051Z] 	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
[2019-05-13T12:14:15.051Z] 	at java.io.FileOutputStream.<init>(FileOutputStream.java:101)
[2019-05-13T12:14:15.051Z] 	at sun.security.tools.keytool.Main.doCommands(Main.java:1194)
[2019-05-13T12:14:15.051Z] 	at sun.security.tools.keytool.Main.run(Main.java:366)
[2019-05-13T12:14:15.051Z] 	at sun.security.tools.keytool.Main.main(Main.java:359)
```

you will need to import the cert and install it the same way for the EE tests, so it would be good to understand what the issue is here - then it should be fairly straightforward to add this to the DAP test too.

Reply:
once you are able to resolve this, we'll probably also need to update the instructions [in the README](https://github.com/cyberark/conjur-api-java#ssl-certificates)

Reply:
It was coming from the update made on openjdk in Dockerfile.text. Once I updated the command in the script from 7 -> 8, it ran fine (: